sdk/storage/azure-storage-blob/src/test/java/com/azure/android/storage/blob/BlobTest.java

```diff
@@ -288,6 +290,32 @@ public void getPropertiesError() {
         assertEquals(404, ex.getStatusCode());
     }
 
+    @Test
+    public void getPropertiesAsync() {
+        // Setup
+        CountDownLatch latch = new CountDownLatch(1);
+
+        // When
+        asyncClient.getBlobProperties(containerName, blobName, new CallbackWithHeader<Void, BlobGetPropertiesHeaders>() {
+                @Override
+                public void onSuccess(Void result, BlobGetPropertiesHeaders header, Response response) {
+                    assertEquals(200, response.code());
+                    latch.countDown();
```

Comment:
Should we check that the header's contents are correct like we check the body contents in `rawRownloadAsync()`?

Reply:
I think we have sync tests that validate the header values, and that logic should be shared between sync and async, so we didn't feel the need to test that redundantly. The async test is more to check that the request goes in and out correctly. Let me know what you think about that approach.

Reply:
Sounds good to me as long as we have good coverage :)